src/Emitter.ts

```diff
@@ -367,14 +892,173 @@ export class Emitter
 
             let parentMod = this._getOrCreateClassNamespace(parent);
 
+            debug(`Emitter._getOrCreateClassNamespace(): pushing ${docletDebugInfo(mod.doclet)} as a child of ${docletDebugInfo(parentMod.doclet)}`);
             mod.doclet.memberof = parentMod.doclet.longname;
             parentMod.children.push(mod);
         }
         else
         {
+            debug(`Emitter._getOrCreateClassNamespace(): no memberof, pushing ${docletDebugInfo(mod.doclet)} as a root`);
             this._treeRoots.push(mod);
         }
 
         return mod;
     }
+
+    /**
+     * Search for a IDocletTreeNode from its longname.
+     * Handles situations where default exported lambda types hold the same longname as the parent module.
+     * @param longname  Longname to search for.
+     * @param filter    Optional filter function that indicates whether a doclet is an appropriate candidate for what we search.
+     */
+    private _getNodeFromLongname(longname: string, filter?: (node: IDocletTreeNode) => boolean): IDocletTreeNode | null
+    {
+        function _debug(msg: string) { /*debug(msg);*/ }
```

Comment:
Some leftover debug code?

Reply:
I left it intentionally, in case sameone needs to debug further the behaviour of this `_getOrCreateClassNamespace()`.
Should I remove it?

Reply:
I remove those detailed debug lines.
If one needs to debug that deep, he/she will do it on his/her own.